### PR TITLE
upgrade to go 1.19

### PR DESCRIPTION
**Pull request by mokpro**
🍏 

- Upgrade app to go 1.19
- Upgrade all the package dependencies
- Fix CI to use `main` branch
- Replace `goling` with `revive`

### CODE REVIEW

File: sqs_client_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build sqsint
 // +build sqsint
```
Comment (packrat386) [inline sqs_client_test.go:2]: You can remove these now

File: worker_client_test.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//go:build redisint
 // +build redisint
```
Comment (packrat386) [inline worker_client_test.go:2]: this too

**Review by packrat386 — DISMISSED** [review]

File: sqs_client_test.go (hunk 1 above)
Comment (mokpro) [reply sqs_client_test.go]: I added this back, as CI shouldn't run these integration tests. 

Can you suggest a better way to ignore these tests?

Comment (packrat386) [reply sqs_client_test.go]: The new tags ( `//go:build sqsint` ) replace the old ones ( `//+build sqsint` ) in go 1.18 (https://tip.golang.org/doc/go1.18#go-build-lines).

You still need the tags, but you don't need both the new and the old.

Comment (mokpro) [reply sqs_client_test.go]: Got it, thanks 👍

**Review by packrat386 — APPROVED** [review]

**Review by acastro2 — APPROVED** [review]

**Pull request merged**